packages/core/custom-elements-lsp/README.md

```diff
@@ -24,14 +24,15 @@ To use this plugin you have a version of typescript as part of the project, loca
         "name": "@genesiscommunitysuccess/custom-elements-lsp",
         "srcRouteFromTSServer": "../../..",
         "designSystemPrefix": "example",
-        "fastEnable": true,
         "parser": {
+          "fastEnable": true,
```

Comment:
nit: not part of this but while checking these changes on a MD viewer noticed that the "# Custom Elements LSP Plugin" block has a `>>` section that could potentially be just `>`